### ISSUE TITLE
removeNuisanceRegressor.sh --compcor_global files in unexpected place

### ISSUE THREAD

**Issue by jdkent** [HBClab/RestingState#99]
### Summary

One of the nuisance regressor files created from the `--compcor_global` option is not being found.

### Actual behavior

```
Running /opt/RestingState/removeNuisanceRegressor.sh ...
/opt/RestingState/ROIs/global.nii.gz is in MNI space
Image Exception : #22 :: ERROR: Could not open image /data/derivatives/rsOut/sub-01/nuisanceRegression/compcor_global/rois/CSF_pve_to_RS_thresh_native
terminate called after throwing an instance of 'RBD_COMMON::BaseException'
/opt/RestingState/removeNuisanceRegressor.sh: line 336:  5825 Aborted                 (core dumped) fslmeants -i "$epiData" -o "$outDir"/rois/mean_CSF_pve_to_RS_thresh_ts.txt -m "$outDir"/rois/CSF_pve_to_RS_thresh_native.nii.gz --eig --order=5
Image Exception : #22 :: ERROR: Could not open image /data/derivatives/rsOut/sub-01/nuisanceRegression/compcor_global/rois/WM_pve_to_RS_thresh_ero_native
terminate called after throwing an instance of 'RBD_COMMON::BaseException'
/opt/RestingState/removeNuisanceRegressor.sh: line 336:  5826 Aborted                 (core dumped) fslmeants -i "$epiData" -o "$outDir"/rois/mean_WM_pve_to_RS_thresh_ero_ts.txt -m "$outDir"/rois/WM_pve_to_RS_thresh_ero_native.nii.gz --eig --order=5
/data/derivatives/rsOut/sub-01/SNR/CSF_pve_to_RS_thresh.nii.gz is in native space
Image Exception : #22 :: ERROR: Could not open image /data/derivatives/rsOut/sub-01/nuisanceRegression/compcor_global/rois/WM_pve_to_RS_thresh_ero_native
terminate called after throwing an instance of 'RBD_COMMON::BaseException'
/opt/RestingState/removeNuisanceRegressor.sh: line 336:  5855 Aborted                 (core dumped) fslmeants -i "$epiData" -o "$outDir"/rois/mean_WM_pve_to_RS_thresh_ero_ts.txt -m "$outDir"/rois/WM_pve_to_RS_thresh_ero_native.nii.gz --eig --order=5
/data/derivatives/rsOut/sub-01/SNR/WM_pve_to_RS_thresh_ero.nii.gz is in native space
```

### Expected behavior

The file should be found and used.

### How to replicate the behavior

use `hbclab/restingstate:v0.3.0`

### Script/Workflow details

Please put URL to code or code here (if not too long).

**Comment by jdkent** [comment]
The `roi` processing steps in [removeNuisanceRegressor.sh](https://github.com/HBClab/RestingState/blob/e5b654f244d271cae934a59831c24268476d48a9/removeNuisanceRegressor.sh#L327) assume the rois are in the `rois` directory, whereas the selected `rois` for `--compcor_global` are in the [SNR dir](https://25-76896762-gh.circle-artifacts.com/0/tmp/data/ds005/derivatives/rsOutCompcorGlobal/sub-01/nuisanceList.txt)

So a fix for how the file paths resolve will fix the issue.

**Comment by jdkent** [comment]
scratch what I said before, `--compcor-global` is trying to do the [mean of the global and eigenvalues of the csf/wm for each roi](https://github.com/HBClab/RestingState/blob/9484ed5f25868f2c58490709a3ba5f77419cd214/removeNuisanceRegressor.sh#L368-L371)

where each roi is the global and and wm/csf masks.

**Comment by mwvoss** [comment]
I think this is because the fslmeants code for compor_global is within a broader loop for each nuisance ROI. Since copying the native space ROI is in the top of the loop, when it's running global, then CSF and WM haven't been copied over yet, but accessing these files is hard-coded in the compcor_global loop. I added a test for the nifit file before running fslmeants command in compcor_global case, and it's testing now.

**Comment by jdkent** [comment]
fixed with #97
